modules/thermal_hydraulics/test/tests/components/heat_transfer_from_heat_structure_1phase/fin_enhancement.i

```diff
@@ -0,0 +1,240 @@
+# This test has 2 pipes, each surrounded by a cylindrical HS:
+#
+#   - pipe1: no fin heat transfer enhancement
+#   - pipe2: fin heat transfer enhancement
+
+diam = 0.01
+area = ${fparse 0.25 * pi * diam^2}
+
+length = 1.0
+n_elems = 10
+
+t_hs = 0.02
+n_elems_radial = 5
+
+rho_inlet = 1359.792245 # @ T = 300 K, p = 1e5 Pa
+vel_inlet = 1.0
+T_inlet = 300
+p_outlet = 1e5
+T_initial_hs = 800
+mfr_inlet = ${fparse rho_inlet * vel_inlet * area}
+
+htc = 100
+
+# Suppose that there are 10 rectangular, 1-mm-thick fins of height 1 cm over the length
+# of the cooled section.
+n_fin = 10
+h_fin = 0.01
```

Comment:
The heat structure is coupled with the flow channel on the inner side, so it'd be more realistic if the fin hight was smaller than the channel diameter.